docs-gen/content/rule_set/data_entry/data_types.md

```diff
@@ -23,3 +23,38 @@ String     | character string           | n/a  | n/a
 ByteBuffer | buffer of bytes (aka BLOB) | n/a | n/a
 
 Datatypes are not specified for ```branch```
+
+
+
+## Data Streams
+
+Data Entries, which describe sensors offering binary streams
+(e.g. cameras), are not supported directly by VSS with a
+dedicated data type. Instead, they are described through the
+meta data about the sensor itself and how to retrieve the
+corresponding data stream.
+
+A camera can be a good example to it. The Data Entry for the camera
+and the corresponding video stream could look like:
+
+```YAML
+- Camera:
+  type: branch
+  description: Information about the camera and how to connect to the video stream
+
+- Camera.IsActive:
+  type: actuator
+  datatype: boolean
+  description: If the camera is active, the client is able to retrieve the video stream
+
+- Camera.URI:
+  type: sensor
+  datatype: string
+  description: URI for retrieving the video stream
+```
+
+In this example, it shows the usage of meta data about
+the status of the sensor. The camera can be set to active through
+the same data entry (`actuator`). A dynamic data entry (`sensor`)
+is used for the URI of the video stream. As long as the camera
+`IsActive`, in this example the `URI` is set and empty otherwise.
```

Comment:
> As long as the camera `IsActive`, in this example the `URI` is set and empty otherwise.

I heard this in the previous call but I did not have time or reason to interject.  I do not think it's required to set the URL to empty if the IsActive is there to give this information?  In any case, if this **is** the behavior of some particular stream, it ought to be in the description I think.  I would not write it here as if it is a required behavior for all streams.

Reply:
>  I heard this in the previous call but I did not have time or reason to interject. I do not think it's required to set the URL to empty if the IsActive is there to give this information? In any case, if this is the behavior of some particular stream, it ought to be in the description I think. I would not write it here as if it is a required behavior for all streams.

Fair enough. It just adds complexity to the example and definitely it's not mandatory. Will remove that one.